testing/classifier.go

```diff
@@ -0,0 +1,48 @@
+package testing
+
+import (
+	"github.com/PlakarKorp/kloset/classifier"
+	"github.com/PlakarKorp/kloset/snapshot/vfs"
+)
+
+func init() {
+	classifier.Register("mock", func() classifier.Backend {
+		return &MockClassifierBackend{name: "mock"}
+	})
+}
```

Comment:
just wondering whether we should use different names for the various pieces instead of using `mock` everywhere.  but it's not a problem, we can fix this later.